server/jetstream_cluster_3_test.go

```diff
@@ -5474,14 +5474,9 @@ func TestJetStreamClusterConsumerMaxDeliveryNumAckPendingBug(t *testing.T) {
 		t.Helper()
 		require_Equal(t, a.Delivered.Consumer, 10)
 		require_Equal(t, a.Delivered.Stream, 10)
-		// If replicated, agreed upon state is used. Otherwise, o.asflr and o.adflr would be skipped ahead for R1.
-		if replicated {
```

Comment:
We used to have a distinction for replicated and non-replicated consumers. Now we are consistent, always reporting the correct data.